javascripts/previewmode.js

```diff
@@ -7,7 +7,29 @@ function htmlToElement(content)
 
 function createDebugFrame()
 {
-    return htmlToElement('<iframe id="mtmDebugFrame" src="about:blank" frameborder="0" style="background-color:#edecec !important; clip: initial !important; display: inline !important; height:33% !important; opacity: 1 !important; visibility:visible !important; padding:0 !important; margin:0 !important; border:0 !important; border-top: 2px solid #fff !important; position:fixed !important; bottom:0 !important; left:0 !important; top:initial !important; width:100% !important; z-index:999999999 !important;"></iframe>');
+    var iframeClass = stickToTop() ? 'mtmStickyTop' : 'mtmStickyBottom';
+    return htmlToElement('<iframe class="'+ iframeClass +'" id="mtmDebugFrame" src="about:blank" frameborder="0" style="background-color:#edecec !important; clip: initial !important; display: inline !important; height:33% !important; opacity: 1 !important; visibility:visible !important; padding:0 !important; margin:0 !important; border:0 !important; border-top: 2px solid #fff !important; position:fixed !important; left:0 !important; width:100% !important; z-index:999999999 !important;min-height: 18rem;"></iframe>');
+}
+
+function stickToTop() {
+    return getCookie('mtmPreviewPosition') === 'top';
```

Comment:
we could potentially remove the entire `getCookie` method and shorten this to `document.cookie.indexOf('mtmPreviewPosition=top') > -1`?

Reply:
Simple and does the work :+1: